### PR TITLE
fix: use default values

### PR DESCRIPTION
**Pull request by jianzhangbjz**
Closes # 
- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
Hi @AlexsJones thanks for reviewing, I have updated it.

File: cmd/auth/add.go [hunk 1]
```diff
@@ -65,15 +65,19 @@ var addCmd = &cobra.Command{
...
+			backend = "openai"
+		} else {
+			if !validBackend(ai.Backends, backend) {
+				color.Red("Error: Backend AI accepted values are '%v'", strings.Join(ai.Backends, ", "))
+				os.Exit(1)
+			}
 		}
 
 		// check if model is not empty
 		if model == "" {
```
Comment (AlexsJones) [inline cmd/auth/add.go:79]: Do you not need to assign `model = "gpt-3.5-turbo"` ?

**Review by AlexsJones — APPROVED** [review]

**Comment by jianzhangbjz** [comment]
Hi @AlexsJones , seems like it needs the manual merge, could you help have a look? Thanks!

**Comment by jianzhangbjz** [comment]
Hi @AlexsJones could you help merge it manually when you get a chance? Thanks!

**Review by AlexsJones — APPROVED** [review]

**Pull request merged**